hphp/hack/hhi/stdlib/builtins_icu_uchar.hhi

```diff
@@ -604,7 +604,7 @@ class IntlChar {
   const int LB_HYPHEN = 0;
   const int LB_IDEOGRAPHIC = 0;
   const int LB_INSEPARABLE = 0;
-  const int LB_INSEPERABLE = 0;
+  const int LB_INSEPARABLE = 0;
```

Comment:
This is imported from PHP API

Reply:
Oops, indeed, looks like they made this mistake and already added the correction (which is also already included here).

I'll drop this change with my next push. (Probably Tuesday or thereabouts.) I imagine that the CI isn't cheap and this will allow me to accumulate fixes.